### DISCUSSION TITLE
Looking for <Setup214_LilyGo_T_Display_S3_Pro.h>

### DISCUSSION THREAD

**Discussion by MikeyMoMo** [Bodmer/TFT_eSPI#3106] [Compatible displays and setup files]
I don't see it in the current version.  Will you have that coming or will it come from TTGO/LilyGo?  It is mentioned in the demo programs from TTGO.  The T-Display S3 setup works perfectly!

 Thanks for the great library.  It is quickly becoming/has become the standard for display driving.  It is surely MY standard library!!!

**Comment by Bodmer** [comment]
Raise a pull request with a tested setup file and I will add it.
